src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -414,7 +375,7 @@ aws_relogin_loop() {
 # Helpers: workdir configuration
 # =========================================================
 
-detect_using_s3_workdir() {
+detect_if_using_s3_workdir() {
```

Comment:
this `detect_if_using_s3_workdir` is doing the same thing as in `setup_workdir` line 393. Also we might need to check and make sure that `USING_S3_WORKDIR=true` is done before `check_cache_db_storage_limit`